docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -66,40 +67,6 @@ sess = onnxruntime.InferenceSession(<path_to_model_file>, options)
 SessionOptions::SetGraphOptimizationLevel(ORT_DISABLE_ALL);
 ```
 
-### Deprecated: Dynamic device type selection
-**Note: This API has been deprecated. Please use the Key-Value mechanism mentioned above to set the 'device-type' option.**
-When ONNX Runtime is built with OpenVINO Execution Provider, a target hardware option needs to be provided. This build time option becomes the default target harware the EP schedules inference on. However, this target may be overriden at runtime to schedule inference on a different hardware as shown below.
-
-Note. This dynamic hardware selection is optional. The EP falls back to the build-time default selection if no dynamic hardware option value is specified.
-
-### Python API
```

Comment:
@sfatimar This python API is deprecated as we know but still works. so let us not remove this.
I tested out, it works.

Reply:
**Example:**  [This works]
import onnxruntime
onnxruntime.capi._pybind_state.set_openvino_device("HETERO:MYRIAD,CPU")